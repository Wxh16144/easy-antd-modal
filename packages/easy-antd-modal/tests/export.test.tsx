describe('export', () => {
  it('should work', async () => {
    const all = await import('easy-antd-modal');

    expect(Object.keys(all)).toMatchSnapshot();
  });

  it.skip('should export all hooks', async () => {
    const hooks = await import('easy-antd-modal/hooks');

    expect(Object.keys(hooks)).toMatchInlineSnapshot(`
      [
        "useBoolean",
        "useLatestFunc",
        "useModalEnhanced",
      ]
    `);
  });

  it.skip('should export all utils', async () => {
    const types = await import('easy-antd-modal/util');

    expect(Object.keys(types)).toMatchInlineSnapshot(`
      [
        "has",
        "omit",
        "isElement",
        "isDOMTypeElement",
      ]
    `);
  });

  describe.skip('mobile', () => {
    it('should work', async () => {
      const all = await import('easy-antd-modal/mobile');

      expect(Object.keys(all)).toMatchSnapshot();
    });
  });
});
