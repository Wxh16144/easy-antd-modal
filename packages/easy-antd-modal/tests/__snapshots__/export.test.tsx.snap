// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`export > should work 1`] = `
[
  "DragModal",
  "Drawer",
  "Modal",
  "useBoolean",
  "useLatestFunc",
  "useModalEnhanced",
  "EasyAntdModalContext",
  "useEasyAntdModal",
  "EasyAntdModalProvider",
  "has",
  "omit",
  "isElement",
  "isDOMTypeElement",
  "useMergeOpen",
  "usePrefixCls",
  "isNil",
  "default",
]
`;
